File: 01-Assignment-02-02-2026/Using_C/program_10.c
/* Program 10: Display the given integer in reverse */

// 7843 --> 3487

#include <stdio.h>
#include <stdlib.h> // get abs

int main() {
    int n,original,reverse = 0,temp;
    printf("Enter a Number : ");
    scanf("%d",&original);

    // for negative no.
    int sign = (original<0)?-1:1;
    n = abs(original);
    while (n>0){
        temp = n % 10;
        reverse = reverse*10 + temp;
        n/=10;
    }
    reverse*=sign;
    printf("Original Number : %d\n",original);
    printf("Reversed Number : %d\n",reverse);
    
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program10.java
/*
Program 10
Java program to display the given integer in reverse manner
*/

import java.util.Scanner;

public class Program10 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter an integer number: ");
        int number = scanner.nextInt();

        int reverse = 0;
        int num = Math.abs(number);

        while (num > 0) {
            reverse = reverse * 10 + num % 10;
            num /= 10;
        }

        if (number < 0) {
            reverse = -reverse;
        }

        System.out.println("Original number : " + number);
        System.out.println("Reversed number : " + reverse);

        scanner.close();
    }
}
